psi4/driver/json_wrapper.py

```diff
@@ -29,15 +29,15 @@
 Runs a JSON input psi file.
 """
 
+import os
+import copy
+import uuid
+
+import psi4
 from psi4.driver import driver
```

Comment:
Is this reordering from PyCharm?

Reply:
No, I just went through and cleared out a lot of unused imports and reorganized into standard modules, external modules, and psi4 relative imports. Then reordered by length of module name. The last is admittedly arbitrary, but imports were already jumbled so it seemed harmless and orderly.